src/sentry/tasks/derive_code_mappings.py

```diff
@@ -43,6 +43,7 @@ def derive_code_mappings(
     set_tag("organization.slug", organization.slug)
     # When you look at the performance page the user is a default column
     set_user({"username": organization.slug})
+    set_tag("project.slug", project.slug)
```

Comment:
Setting the tag sooner means that we increase the chances of an error containing the tag (e.g. if an error occurred before reaching `report_project_codemappings`).